hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -675,6 +675,8 @@ public final class OzoneConfigKeys {
   public static final String HDDS_SCM_CLIENT_FAILOVER_MAX_RETRY =
       "hdds.scmclient.failover.max.retry";
 
+  public static final String OZONE_XCEIVER_METRICS_PERCENTILES_INTERVALS_SECONDS_KEY =
+      "ozone.xceiver.metrics.percentiles.intervals.seconds";
```

Comment:
`xceiver` is an "internal" name, it may be very obscure for admins.  No other configuration key mentions it.

`XceiverClientMetrics` is described as `"Storage Container Client Metrics"`.

Reply:
Thanks @adoroszlai , I think it can be rename to OZONE_STORANG_CONTAINER_CLIENT_METRICS_PERCENTILES_INTERVALS_SECONDS_KEY, what do you think?

Reply:
@adoroszlai. Unfortunately the metric name is already `XceiverClientMetrics`
https://github.com/apache/ozone/blob/e89fee375efb1aaeb59978c1c708b6808e38b267/hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientMetrics.java#L37
So `ozone.xceiver.metrics.xxx` seems a more direct name. How do you think?